pyproject.toml

```diff
@@ -0,0 +1,85 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "optax"
+dynamic = ["version"]
+description = "A gradient processing and optimisation library in JAX."
+readme = "README.md"
+license = { file = "LICENSE" }
+requires-python = ">=3.8"
+authors = [
+    {name = "DeepMind", email = "optax-dev@google.com"},
+]
+keywords = [
+    "python", 
+    "machine learning", 
+    "reinforcement-learning"
+]
+classifiers = [
+    "Environment :: Console",
+    "Programming Language :: Python",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Intended Audience :: Science/Research",
+    "Development Status :: 5 - Production/Stable",
```

Comment:
Perhaps we would be better off writing "4" here?

Since our version number is `0.X.Y`, technically we are reserving the ability to make breaking API changes between versions.